core/Dockerfile

```diff
@@ -23,5 +22,5 @@ EXPOSE 8001 8024
 
 ENV MAILMAN_CONFIG_FILE /etc/mailman.cfg
 
-ENTRYPOINT ["/opt/run.sh"]
-CMD ["/usr/local/bin/master"]
+ENTRYPOINT ["docker-entrypoint.sh"]
+CMD ["master"]
```

Comment:
Yes this is usually different in different places. The idea is basically that some login shell like BASH (which we are using for entrypoint) does set the PATH variable properly and hence most of the times people can rely on it, but not always because `exec`s from `sh` for example don't always have PATH set.

I agree it is just a matter of convention but I like to keep the full path.